src/app/core/services/artist/artist.service.ts

```diff
@@ -1,5 +1,5 @@
 import { Injectable } from '@angular/core';
-import { Artist } from '../../../models/artist.model';
+import { Artist } from 'src/app/models/artist.model';
```

Comment:
i think the other way of importing this was correct

Reply:
changed